features/performance/manifests/templates/01-machine-config-worker-rt-kernel.yaml.in

```diff
@@ -3,7 +3,7 @@ kind: MachineConfig
 metadata:
     labels:
         machineconfiguration.openshift.io/role: worker-rt
-    name: 11-rt-kernel-worker-rt
+    name: worker-rt-kernel
```

Comment:
@cynepco3hahue don't  we need the `11-` here in the name. i thought that impacted the execution order of the MC on the nodes.

Reply:
@davidvossel hm I named the file with `01-` prefix, so on create, `kubectl` should apply it before all others configurations
As far as I can see `machine-config` daemon, run standard reconcile loop, so if we will assume that we added a number of configurations, it will add them to the controller queue and after reconcile it one by one, so I assume only the file name is important
WDT?

Reply:
i don't think the manifest file name means anything. We're posting this MC to the cluster right? i thought MCO was keying off the object name, or something like that.

Reply:
hm looks like you are right, https://github.com/openshift/machine-config-operator/blob/a27cde5fc36f1bf76710affbbd58e1f0363d9009/pkg/apis/machineconfiguration.openshift.io/v1/helpers.go#L21

and it will be better to pause the machine-config until we create all resources to avoid races

Reply:
done